src/app/language-selection/language-selection.component.ts

```diff
@@ -0,0 +1,35 @@
+import {Component, OnInit} from '@angular/core';
+import {TranslateService} from '@ngx-translate/core';
+import {MediaChange, ObservableMedia} from '@angular/flex-layout';
+
+export const supportedLanguages = [
+  {code: 'en', displayName: 'English'},
+  {code: 'bg', displayName: 'Bulgarian'},
+];
+
+@Component({
+  selector: 'app-language-selection',
+  templateUrl: './language-selection.component.html',
+  styleUrls: ['./language-selection.component.scss']
+})
+export class LanguageSelectionComponent implements OnInit {
+
+  languages = supportedLanguages;
```

Comment:
please sort the languages alphabetically according to display name (just in case)